apps/web/composables/useProductRecommended/index.ts

```diff
@@ -1 +1,2 @@
 export * from './useProductRecommended';
+// export * from './types';
```

Comment:
Why is this commented?

Reply:
You get the error mentioned in the PR:
```
composables/useProductRecommended/index.ts:2:1 - error TS2308: Module './useProductRecommended' has already exported a member named 'useProductRecommended'. Consider explicitly re-exporting to resolve the ambiguity.

2 export * from './types';
```
I wanted to make a note that it's there, but solve it separately.